TheJumpCloud/support
# Sa 1375 get jc object dynamic search some endpoints

## Issues
* [SA-1375](https://jumpcloud.atlassian.net/browse/SA-1375) - SA-1375-Get-JCObject dynamic search endpoints

## What does this solve?
Fixes a bug where radius_server and cloud directory endpoints output incorrect values
## Is there anything particularly tricky?

## How should this be tested?
1. Run `get-jcRadiusServer -Name "Test"`. This search uses a searchTerm which will output any radius server that have the name ex. RadiusTest and JCTest will get outputted if you search for the name Test

- Multiple Servers "Name Search" | Should return "Matches" (non-case sensitive)
- Name Search with no matching searches should return nothing

3. Validate g_suite or office_365 `Add-JCAssociation`   ex. `Add-JCAssociation -Type g_suite -Name "test" -TargetType user  -TargetName "tester"`. This should add the user tester to the g_suite directory test

- Multiple Servers "Name Search" | Should return "Matches" (non-case sensitive)
- Name Search with no matching searches should return nothing
